praw/objector.py

```diff
@@ -66,6 +85,35 @@ def objectify(self, data):
                 .issubset(set(data.keys()))):
             parser = self.parsers['ModmailConversation']
             return parser.parse(data, self._reddit)
+        elif isinstance(data, dict) and 'actionTypeId' in data.keys():
```

Comment:
Given that all of these start with `isinstance(data, dict)` I think we should have just one case for all of those, and then refactor that to call a private method like `_objectify_dict` as this method has grown rather large.